components/organisms/o-footer.vue

```diff
@@ -95,11 +95,11 @@ export default {
           name: 'About us',
           children: [
             {
-              name: 'About us (Magento CMS)',
+              name: 'About us',
```

Comment:
Wiem, że to powinno być wyłapane w CR odnośnie footera, ale dopiero teraz zauważyłem. Name w tym obiekcie, trzeba wrzucić w funkcję do tłumaczeń

Reply:
This goes already through translation function:

`:label="$t(link.name)"`

in this line https://github.com/DivanteLtd/vsf-capybara/pull/129/files#diff-a374d3c1a60eceac552624122a936901R12

Reply:
Right, sorry :)